app/mailers/project_mailer.rb

```diff
@@ -5,4 +5,11 @@ def created_email(user, project)
     body = "#{user.name_and_email} just created a new project #{project.name}"
     mail(to: address, subject: subject, body: body)
   end
+
+  def deleted_email(user, project)
+    address = ENV['PROJECT_DELETED_NOTIFY_ADDRESS'].presence || ENV['PROJECT_CREATED_NOTIFY_ADDRESS']
```

Comment:
Moved it to config/application.rb

To be consistent also moved the ENV['PROJECT_CREATED_NOTIFY_ADDRESS'] to a config value as well.